From 01cd91054302432ef31f45fe5fa48b71f9ad29fc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Thu, 2 May 2024 13:57:22 +0200
Subject: [PATCH 0860/1695] dts: stm32: enable support for IWDG1 on
 stm32mp215f-dk board
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add IWDG1 node in stm32mp215f-dk board's device-tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1acf3a9cada4c4158a39530a261fa164bcfd2d90 dts: stm32: enable support for IWDG1 on stm32mp215f-dk board]
Change-Id: Ie6b51a88ff5149d3ddcb3e63c9b03aec2e81b876
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/377759
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index 265971c15..2da2940c8 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -53,6 +53,11 @@
 	status = "okay";
 };
 
+&iwdg1 {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &pka {
 	status = "okay";
 };
-- 
2.47.2

